code/app/commodity/server.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"github.com/brahma-adshonor/micro_service/code/app/commodity/model"
+
+	"fmt"
+	"net/http"
+)
+
+func commodityHandler(w http.ResponseWriter, r *http.Request) {
+	type m model.Model
```

Comment:
type a response instead of original Model, then marshal it and write to body

Reply:
got it.  try to finish it today

Reply:
done